### PR TITLE
alipay: the accessSync interface needs to check if it is successful.

### PR DESCRIPTION
**Pull request by bofeng-song**

### CODE REVIEW

File: common/engine/AssetManager.js [hunk 1]
```diff
@@ -234,7 +234,7 @@ function downloadBundle (nameOrUrl, options, onComplete) {
                     let sys = cc.sys;
                     if (sys.platform === sys.ALIPAY_GAME && sys.os === sys.OS_ANDROID) {
                         let resPath = unzipPath + 'res/';
-                        if (fs.accessSync({path: resPath})) {
+                        if (fs.accessSync({path: resPath}).success) {
```
Comment (PPpro) [inline common/engine/AssetManager.js:237]: the usage is conflit with their current docs https://opendocs.alipay.com/mini/api/025027

Comment (bofeng-song) [reply common/engine/AssetManager.js:237]: according the docs
https://www.yuque.com/zhifubaoxiaoyouxi-zromu/ts2ry9/4e26f2203cdb4c74f80c7fd37382ad0f

Comment (PPpro) [reply common/engine/AssetManager.js:237]: this is the earliest docs they provided several years ago, is it still valid for now ?

Comment (bofeng-song) [reply common/engine/AssetManager.js:237]: They said to use the modification suggestions they provided.

Comment (bofeng-song) [reply common/engine/AssetManager.js:237]: They said we can refer to the usage method at https://www.yuque.com/zhifubaoxiaoyouxi-zromu/ts2ry9/4e26f2203cdb4c74f80c7fd37382ad0f.

**Review by PPpro — APPROVED** [review]

**Pull request merged**